oml/models/resnet.py

```diff
@@ -31,11 +31,15 @@ class ResnetExtractor(IExtractor):
     }
 
     pretrained_models = {
-        "resnet50_moco_v2": (
-            "https://dl.fbaipublicfiles.com/moco/moco_checkpoints/moco_v2_800ep/moco_v2_800ep_pretrain.pth.tar",
-            "a04e12f8",
-            None,
-        )
+        "resnet50_moco_v2": {
+            "url": "https://dl.fbaipublicfiles.com/moco/moco_checkpoints/moco_v2_800ep/moco_v2_800ep_pretrain.pth.tar",
+            "hash": "a04e12f8",
+            "fname": "moco_v2_800ep_pretrain.pth.tar",
```

Comment:
Does `fname` become mandatory? No `None` anymore?

Reply:
It's not mandatory, but I think having `fname` is good because of 2 reasons:
1) names can overlap between different pretrained checkpoints, since the file name on a disk and the name in URL may be different 
2) because of the reason above, it's also easier to understand where is the ckpt file related to the pretrained ckpt that you downloaded

Reply:
Just  a question